docs/articles/documentation/reference/test-api/selector/withexacttext.md

```diff
@@ -28,4 +28,6 @@ Selector('.container').withExactText('foo');
 
 ## Notes
 
-`withExactText` selects the element that contains the specified text and its ancestors (if they do not contain any other text). See an example for [withText](withtext.md#notes).
\ No newline at end of file
+`withExactText` selects the element that contains the specified text and its ancestors (if they do not contain any other text). See an example for [withText](withtext.md#notes).
+
+If you want to target elements, containing special HTML characters, refer to [this section](../../../guides/basic-guides/select-page-elements.md#select-elements-containing-special-characters).
```

Comment:
> `If you want to target elements, containing special HTML characters, refer to [this section](../../../guides/basic-guides/select-page-elements.md#select-elements-containing-special-characters).`

This is why our correctors don't welcome `-ing`s. Elements that contain? You want while containing? It's ambiguous in English, it works only in Russian.

We said 'that contains' in the previous sentence, so here we can say 'with special HTML characters'.

Reply:
edited